2025-05-26 13:23:41.912 UTC | INFO     | apscheduler.scheduler     | Adding job tentatively -- it will be properly scheduled when the scheduler starts
2025-05-26 13:23:41.912 UTC | INFO     | apscheduler.scheduler     | Added job "refresh_active_mission_cache" to job store "default"
2025-05-26 13:23:41.918 UTC | INFO     | apscheduler.scheduler     | Scheduler started
2025-05-26 13:23:41.918 UTC | INFO     | app.app                   | APScheduler started for background cache refresh.
2025-05-26 13:23:41.918 UTC | INFO     | app.app                   | BACKGROUND TASK: Starting proactive cache refresh for active real-time missions.
2025-05-26 13:23:41.918 UTC | INFO     | app.app                   | BACKGROUND TASK: Proactive cache refresh for active real-time missions completed.
2025-05-26 13:23:43.041 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.077 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.086 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.112 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.127 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.145 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.164 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.183 UTC | INFO     | app.app                   | Attempting remote load for ais (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.197 UTC | INFO     | app.app                   | Attempting remote load for errors (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:43.272 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for power (m203). Will try next path.
2025-05-26 13:23:43.282 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.282 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for ctd (m203). Will try next path.
2025-05-26 13:23:43.282 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for vr2c (m203). Will try next path.
2025-05-26 13:23:43.282 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for weather (m203). Will try next path.
2025-05-26 13:23:43.282 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for waves (m203). Will try next path.
2025-05-26 13:23:43.282 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for solar (m203). Will try next path.
2025-05-26 13:23:43.300 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.320 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.337 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.357 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.377 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.388 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for ais (m203). Will try next path.
2025-05-26 13:23:43.388 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for fluorometer (m203). Will try next path.
2025-05-26 13:23:43.389 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for errors (m203). Will try next path.
2025-05-26 13:23:43.405 UTC | INFO     | app.app                   | Attempting remote load for ais (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.424 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.433 UTC | INFO     | app.app                   | Attempting remote load for errors (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:43.547 UTC | INFO     | app.app                   | Successfully loaded errors for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:43.547 UTC | INFO     | app.app                   | CACHE STORE: Storing errors for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:43.567 UTC | INFO     | app.app                   | Successfully loaded vr2c for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:43.570 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:43.594 UTC | INFO     | app.app                   | Successfully loaded power for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:43.594 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:43.611 UTC | INFO     | app.app                   | Successfully loaded waves for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:43.611 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:43.652 UTC | INFO     | app.app                   | Successfully loaded solar for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:43.662 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:43.746 UTC | INFO     | app.app                   | Successfully loaded ais for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:43.746 UTC | INFO     | app.app                   | CACHE STORE: Storing ais for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:43.792 UTC | INFO     | app.app                   | Successfully loaded weather for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:43.792 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:44.048 UTC | INFO     | app.app                   | Successfully loaded fluorometer for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:44.050 UTC | INFO     | app.app                   | CACHE STORE: Storing fluorometer for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:44.218 UTC | INFO     | app.app                   | Successfully loaded ctd for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:44.218 UTC | INFO     | app.app                   | CACHE STORE: Storing ctd for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:45.840 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:45.858 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:45.882 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:45.907 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:45.927 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:45.932 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m203.
2025-05-26 13:23:45.944 UTC | INFO     | app.app                   | Attempting remote load for telemetry (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:45.981 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for ctd (m203). Will try next path.
2025-05-26 13:23:45.981 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for weather (m203). Will try next path.
2025-05-26 13:23:45.994 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.007 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.011 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for solar (m203). Will try next path.
2025-05-26 13:23:46.011 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for telemetry (m203). Will try next path.
2025-05-26 13:23:46.011 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for waves (m203). Will try next path.
2025-05-26 13:23:46.015 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for power (m203). Will try next path.
2025-05-26 13:23:46.032 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.050 UTC | INFO     | app.app                   | Attempting remote load for telemetry (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.072 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.093 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.289 UTC | INFO     | app.app                   | Successfully loaded power for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:46.289 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:46.323 UTC | INFO     | app.app                   | Successfully loaded waves for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:46.348 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:46.350 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:46.393 UTC | INFO     | app.app                   | Successfully loaded solar for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:46.411 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:46.415 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:46.482 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for vr2c (m203). Will try next path.
2025-05-26 13:23:46.507 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.536 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m203-SV3-1070 (C34164NS) for fluorometer (m203). Will try next path.
2025-05-26 13:23:46.562 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m203, remote folder: m203-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:46.658 UTC | INFO     | app.app                   | Successfully loaded telemetry for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:46.660 UTC | INFO     | app.app                   | CACHE STORE: Storing telemetry for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:46.670 UTC | INFO     | app.app                   | Inferred location for mission m203: Lat=44.661656666666666, Lon=-63.55683833333333
2025-05-26 13:23:46.670 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 13:23:46.698 UTC | INFO     | app.app                   | Successfully loaded vr2c for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:46.698 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:46.888 UTC | INFO     | app.app                   | Successfully loaded weather for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:46.893 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:47.349 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.661656666666666&longitude=-63.55683833333333&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:47.349 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.661656666666666, lon=-63.55683833333333. Falling back to general forecast.
2025-05-26 13:23:47.349 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 13:23:48.998 UTC | INFO     | app.app                   | Successfully loaded ctd for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:49.002 UTC | INFO     | app.app                   | CACHE STORE: Storing ctd for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:50.221 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.243 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.263 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.285 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.309 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.329 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.354 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.373 UTC | INFO     | app.app                   | Attempting remote load for ais (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.394 UTC | INFO     | app.app                   | Attempting remote load for errors (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:50.455 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for power (m204). Will try next path.
2025-05-26 13:23:50.455 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for ctd (m204). Will try next path.
2025-05-26 13:23:50.455 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for vr2c (m204). Will try next path.
2025-05-26 13:23:50.472 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.496 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.519 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.525 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for weather (m204). Will try next path.
2025-05-26 13:23:50.525 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for waves (m204). Will try next path.
2025-05-26 13:23:50.527 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for fluorometer (m204). Will try next path.
2025-05-26 13:23:50.529 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for solar (m204). Will try next path.
2025-05-26 13:23:50.529 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for ais (m204). Will try next path.
2025-05-26 13:23:50.549 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.574 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.577 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for errors (m204). Will try next path.
2025-05-26 13:23:50.600 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.627 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.647 UTC | INFO     | app.app                   | Attempting remote load for ais (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.672 UTC | INFO     | app.app                   | Attempting remote load for errors (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:50.709 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for vr2c (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Vemco%20VR2c%20Status.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.710 UTC | WARNING  | app.app                   | No usable remote data found for vr2c (m204). Falling back to default local.
2025-05-26 13:23:50.710 UTC | INFO     | app.app                   | Attempting fallback local load for vr2c (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:50.739 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:50.739 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for ctd (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Seabird%20CTD%20Records%20with%20D.O..csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.739 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for power (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Amps%20Power%20Summary%20Report.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.739 UTC | WARNING  | app.app                   | No usable remote data found for ctd (m204). Falling back to default local.
2025-05-26 13:23:50.743 UTC | INFO     | app.app                   | Attempting fallback local load for ctd (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:50.929 UTC | INFO     | app.app                   | CACHE STORE: Storing ctd for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:50.929 UTC | WARNING  | app.app                   | No usable remote data found for power (m204). Falling back to default local.
2025-05-26 13:23:50.929 UTC | INFO     | app.app                   | Attempting fallback local load for power (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:50.954 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:50.960 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for fluorometer (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Fluorometer%20Samples%202.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.960 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for weather (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Weather%20Records%202.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.960 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for solar (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Amps%20Solar%20Input%20Port%20Report.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.960 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for ais (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/AIS%20Report.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.960 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for waves (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/GPS%20Waves%20Sensor%20Data.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:50.964 UTC | WARNING  | app.app                   | No usable remote data found for fluorometer (m204). Falling back to default local.
2025-05-26 13:23:50.964 UTC | INFO     | app.app                   | Attempting fallback local load for fluorometer (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:51.051 UTC | INFO     | app.app                   | CACHE STORE: Storing fluorometer for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:51.051 UTC | WARNING  | app.app                   | No usable remote data found for weather (m204). Falling back to default local.
2025-05-26 13:23:51.051 UTC | INFO     | app.app                   | Attempting fallback local load for weather (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:51.087 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:51.089 UTC | WARNING  | app.app                   | No usable remote data found for solar (m204). Falling back to default local.
2025-05-26 13:23:51.089 UTC | INFO     | app.app                   | Attempting fallback local load for solar (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:51.113 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:51.113 UTC | WARNING  | app.app                   | No usable remote data found for ais (m204). Falling back to default local.
2025-05-26 13:23:51.113 UTC | INFO     | app.app                   | Attempting fallback local load for ais (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:51.143 UTC | INFO     | app.app                   | CACHE STORE: Storing ais for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:51.143 UTC | WARNING  | app.app                   | No usable remote data found for waves (m204). Falling back to default local.
2025-05-26 13:23:51.143 UTC | INFO     | app.app                   | Attempting fallback local load for waves (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:51.165 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:51.165 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for errors (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Vehicle%20Error%20Report.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:51.165 UTC | WARNING  | app.app                   | No usable remote data found for errors (m204). Falling back to default local.
2025-05-26 13:23:51.165 UTC | INFO     | app.app                   | Attempting fallback local load for errors (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:51.182 UTC | INFO     | app.app                   | CACHE STORE: Storing errors for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:51.787 UTC | INFO     | app.app                   | Successfully loaded fluorometer for mission m203 from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)
2025-05-26 13:23:51.787 UTC | INFO     | app.app                   | CACHE STORE: Storing fluorometer for m203 (from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:23:51.867 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:51.886 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:51.904 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:51.921 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:51.946 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:51.946 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m204.
2025-05-26 13:23:51.963 UTC | INFO     | app.app                   | Attempting remote load for telemetry (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:52.022 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for solar (m204). Will try next path.
2025-05-26 13:23:52.039 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.043 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for ctd (m204). Will try next path.
2025-05-26 13:23:52.045 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for weather (m204). Will try next path.
2025-05-26 13:23:52.045 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for power (m204). Will try next path.
2025-05-26 13:23:52.045 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for waves (m204). Will try next path.
2025-05-26 13:23:52.064 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.084 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.105 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.126 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.130 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for telemetry (m204). Will try next path.
2025-05-26 13:23:52.148 UTC | INFO     | app.app                   | Attempting remote load for telemetry (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.179 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for solar (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Amps%20Solar%20Input%20Port%20Report.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.179 UTC | WARNING  | app.app                   | No usable remote data found for solar (m204). Falling back to default local.
2025-05-26 13:23:52.181 UTC | INFO     | app.app                   | Attempting fallback local load for solar (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:52.195 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:52.220 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for power (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Amps%20Power%20Summary%20Report.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.220 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for ctd (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Seabird%20CTD%20Records%20with%20D.O..csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.220 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for waves (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/GPS%20Waves%20Sensor%20Data.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.220 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for weather (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Weather%20Records%202.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.220 UTC | WARNING  | app.app                   | No usable remote data found for power (m204). Falling back to default local.
2025-05-26 13:23:52.220 UTC | INFO     | app.app                   | Attempting fallback local load for power (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:52.233 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:52.251 UTC | WARNING  | app.app                   | No usable remote data found for ctd (m204). Falling back to default local.
2025-05-26 13:23:52.253 UTC | INFO     | app.app                   | Attempting fallback local load for ctd (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:52.455 UTC | INFO     | app.app                   | CACHE STORE: Storing ctd for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:52.506 UTC | WARNING  | app.app                   | No usable remote data found for waves (m204). Falling back to default local.
2025-05-26 13:23:52.506 UTC | INFO     | app.app                   | Attempting fallback local load for waves (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:52.516 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:52.531 UTC | WARNING  | app.app                   | No usable remote data found for weather (m204). Falling back to default local.
2025-05-26 13:23:52.531 UTC | INFO     | app.app                   | Attempting fallback local load for weather (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:52.552 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:52.586 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:52.604 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:23:52.611 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for telemetry (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Telemetry%206%20Report%20by%20WGMS%20Datetime.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.624 UTC | WARNING  | app.app                   | No usable remote data found for telemetry (m204). Falling back to default local.
2025-05-26 13:23:52.628 UTC | INFO     | app.app                   | Attempting fallback local load for telemetry (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:52.719 UTC | INFO     | app.app                   | CACHE STORE: Storing telemetry for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:52.735 UTC | INFO     | app.app                   | Inferred location for mission m204: Lat=44.47753281138516, Lon=-63.39545485439191
2025-05-26 13:23:52.735 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 13:23:52.785 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for fluorometer (m204). Will try next path.
2025-05-26 13:23:52.785 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m204-SV3-1070 (C34164NS) for vr2c (m204). Will try next path.
2025-05-26 13:23:52.807 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.827 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m204, remote folder: m204-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:23:52.888 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for vr2c (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Vemco%20VR2c%20Status.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.889 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for fluorometer (m204): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m204-SV3-1070%20(C34164NS)/Fluorometer%20Samples%202.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:52.889 UTC | WARNING  | app.app                   | No usable remote data found for vr2c (m204). Falling back to default local.
2025-05-26 13:23:52.889 UTC | INFO     | app.app                   | Attempting fallback local load for vr2c (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:52.894 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:53.055 UTC | WARNING  | app.app                   | No usable remote data found for fluorometer (m204). Falling back to default local.
2025-05-26 13:23:53.055 UTC | INFO     | app.app                   | Attempting fallback local load for fluorometer (mission: m204) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:23:53.122 UTC | INFO     | app.app                   | CACHE STORE: Storing fluorometer for m204 (from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:23:53.271 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.47753281138516&longitude=-63.39545485439191&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 13:23:53.271 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.47753281138516, lon=-63.39545485439191. Falling back to general forecast.
2025-05-26 13:23:53.271 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 13:26:26.606 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.628 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.645 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.668 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.689 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.700 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.726 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.741 UTC | INFO     | app.app                   | Attempting remote load for ais (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:26.957 UTC | INFO     | app.app                   | Attempting remote load for errors (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:27.054 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for vr2c (m199). Will try next path.
2025-05-26 13:26:27.055 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for weather (m199). Will try next path.
2025-05-26 13:26:27.057 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for waves (m199). Will try next path.
2025-05-26 13:26:27.057 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for power (m199). Will try next path.
2025-05-26 13:26:27.057 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for ctd (m199). Will try next path.
2025-05-26 13:26:27.078 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.096 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.130 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.144 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.177 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.189 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for ais (m199). Will try next path.
2025-05-26 13:26:27.189 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for fluorometer (m199). Will try next path.
2025-05-26 13:26:27.189 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for errors (m199). Will try next path.
2025-05-26 13:26:27.189 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for solar (m199). Will try next path.
2025-05-26 13:26:27.205 UTC | INFO     | app.app                   | Attempting remote load for ais (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.236 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.252 UTC | INFO     | app.app                   | Attempting remote load for errors (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.284 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:27.321 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for ctd (m199): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m199-SV3-1070%20(C34164NS)/Seabird%20CTD%20Records%20with%20D.O..csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:26:27.321 UTC | WARNING  | app.app                   | No usable remote data found for ctd (m199). Falling back to default local.
2025-05-26 13:26:27.321 UTC | INFO     | app.app                   | Attempting fallback local load for ctd (mission: m199) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:26:27.321 UTC | INFO     | app.core.loaders          | File not found at local path: C:\Users\ty225269\Documents\Python Playground\Data\m199\Seabird CTD Records with D.O..csv. Error: [Errno 2] No such file or directory: 'C:\\Users\\ty225269\\Documents\\Python Playground\\Data\\m199\\Seabird CTD Records with D.O..csv'
2025-05-26 13:26:27.321 UTC | INFO     | app.app                   | Fallback local file not found for ctd (m199) at C:\Users\ty225269\Documents\Python Playground\Data\m199
2025-05-26 13:26:27.347 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for fluorometer (m199): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m199-SV3-1070%20(C34164NS)/Fluorometer%20Samples%202.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:26:27.351 UTC | WARNING  | app.app                   | No usable remote data found for fluorometer (m199). Falling back to default local.
2025-05-26 13:26:27.351 UTC | INFO     | app.app                   | Attempting fallback local load for fluorometer (mission: m199) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:26:27.351 UTC | INFO     | app.core.loaders          | File not found at local path: C:\Users\ty225269\Documents\Python Playground\Data\m199\Fluorometer Samples 2.csv. Error: [Errno 2] No such file or directory: 'C:\\Users\\ty225269\\Documents\\Python Playground\\Data\\m199\\Fluorometer Samples 2.csv'
2025-05-26 13:26:27.351 UTC | INFO     | app.app                   | Fallback local file not found for fluorometer (m199) at C:\Users\ty225269\Documents\Python Playground\Data\m199
2025-05-26 13:26:27.410 UTC | INFO     | app.app                   | Successfully loaded errors for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:27.410 UTC | INFO     | app.app                   | CACHE STORE: Storing errors for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:27.495 UTC | INFO     | app.app                   | Successfully loaded vr2c for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:27.495 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:27.581 UTC | INFO     | app.app                   | Successfully loaded waves for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:27.581 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:27.606 UTC | INFO     | app.app                   | Successfully loaded power for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:27.606 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:27.717 UTC | INFO     | app.app                   | Successfully loaded solar for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:27.717 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:27.886 UTC | INFO     | app.app                   | Successfully loaded weather for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:27.886 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:28.493 UTC | INFO     | app.app                   | Successfully loaded ais for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:28.495 UTC | INFO     | app.app                   | CACHE STORE: Storing ais for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:29.098 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.115 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.144 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.147 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m199.
2025-05-26 13:26:29.171 UTC | INFO     | app.app                   | Attempting remote load for telemetry (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.192 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.206 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.247 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for ctd (m199). Will try next path.
2025-05-26 13:26:29.266 UTC | INFO     | app.app                   | Attempting remote load for ctd (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.276 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for waves (m199). Will try next path.
2025-05-26 13:26:29.291 UTC | INFO     | app.app                   | Attempting remote load for waves (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.300 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for weather (m199). Will try next path.
2025-05-26 13:26:29.300 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for power (m199). Will try next path.
2025-05-26 13:26:29.301 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for telemetry (m199). Will try next path.
2025-05-26 13:26:29.301 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for solar (m199). Will try next path.
2025-05-26 13:26:29.324 UTC | INFO     | app.app                   | Attempting remote load for weather (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.341 UTC | INFO     | app.app                   | Attempting remote load for power (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.359 UTC | INFO     | app.app                   | Attempting remote load for telemetry (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.374 UTC | INFO     | app.app                   | Attempting remote load for solar (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.411 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for ctd (m199): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m199-SV3-1070%20(C34164NS)/Seabird%20CTD%20Records%20with%20D.O..csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:26:29.411 UTC | WARNING  | app.app                   | No usable remote data found for ctd (m199). Falling back to default local.
2025-05-26 13:26:29.411 UTC | INFO     | app.app                   | Attempting fallback local load for ctd (mission: m199) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:26:29.411 UTC | INFO     | app.core.loaders          | File not found at local path: C:\Users\ty225269\Documents\Python Playground\Data\m199\Seabird CTD Records with D.O..csv. Error: [Errno 2] No such file or directory: 'C:\\Users\\ty225269\\Documents\\Python Playground\\Data\\m199\\Seabird CTD Records with D.O..csv'
2025-05-26 13:26:29.411 UTC | INFO     | app.app                   | Fallback local file not found for ctd (m199) at C:\Users\ty225269\Documents\Python Playground\Data\m199
2025-05-26 13:26:29.429 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.499 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for vr2c (m199). Will try next path.
2025-05-26 13:26:29.517 UTC | INFO     | app.app                   | Attempting remote load for vr2c (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.535 UTC | INFO     | app.app                   | Successfully loaded waves for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:29.535 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:29.571 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_realtime_missions
2025-05-26 13:26:29.608 UTC | INFO     | app.app                   | Successfully loaded power for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:29.608 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:29.641 UTC | INFO     | app.app                   | Successfully loaded solar for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:29.641 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:29.674 UTC | INFO     | app.app                   | File not found in realtime path (expected for past missions): http://129.173.20.180:8086/output_realtime_missions/m199-SV3-1070 (C34164NS) for fluorometer (m199). Will try next path.
2025-05-26 13:26:29.692 UTC | INFO     | app.app                   | Attempting remote load for fluorometer (mission: m199, remote folder: m199-SV3-1070 (C34164NS)) from base: http://129.173.20.180:8086/output_past_missions
2025-05-26 13:26:29.700 UTC | INFO     | app.app                   | Successfully loaded vr2c for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:29.700 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:29.889 UTC | INFO     | app.app                   | Successfully loaded weather for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:29.895 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:29.959 UTC | WARNING  | app.app                   | Remote load attempt from http://129.173.20.180:8086/output_past_missions failed for fluorometer (m199): Client error '404 Not Found' for url 'http://129.173.20.180:8086/output_past_missions/m199-SV3-1070%20(C34164NS)/Fluorometer%20Samples%202.csv'
For more information check: https://httpstatuses.com/404
2025-05-26 13:26:29.961 UTC | WARNING  | app.app                   | No usable remote data found for fluorometer (m199). Falling back to default local.
2025-05-26 13:26:29.961 UTC | INFO     | app.app                   | Attempting fallback local load for fluorometer (mission: m199) from C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:26:29.961 UTC | INFO     | app.core.loaders          | File not found at local path: C:\Users\ty225269\Documents\Python Playground\Data\m199\Fluorometer Samples 2.csv. Error: [Errno 2] No such file or directory: 'C:\\Users\\ty225269\\Documents\\Python Playground\\Data\\m199\\Fluorometer Samples 2.csv'
2025-05-26 13:26:29.961 UTC | INFO     | app.app                   | Fallback local file not found for fluorometer (m199) at C:\Users\ty225269\Documents\Python Playground\Data\m199
2025-05-26 13:26:30.143 UTC | INFO     | app.app                   | Successfully loaded telemetry for mission m199 from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)
2025-05-26 13:26:30.144 UTC | INFO     | app.app                   | CACHE STORE: Storing telemetry for m199 (from Remote: http://129.173.20.180:8086/output_past_missions/m199-SV3-1070 (C34164NS)) into cache.
2025-05-26 13:26:30.156 UTC | INFO     | app.app                   | Inferred location for mission m199: Lat=44.5873, Lon=-63.48034500000001
2025-05-26 13:26:30.156 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 13:26:30.666 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.5873&longitude=-63.48034500000001&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 13:26:30.670 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.5873, lon=-63.48034500000001. Falling back to general forecast.
2025-05-26 13:26:30.670 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 13:27:27.895 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.895 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.903 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.903 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.903 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.903 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.903 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.903 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ais (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:27.903 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for errors (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:28.858 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:28.908 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:29.075 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m204.
2025-05-26 13:27:29.075 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for telemetry (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:29.103 UTC | INFO     | app.app                   | Inferred location for mission m204: Lat=44.47753281138516, Lon=-63.39545485439191
2025-05-26 13:27:29.103 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 13:27:29.122 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:29.149 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:29.170 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:29.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:29.215 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m204). Returning cached data from Local (Default Fallback): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:27:29.743 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.47753281138516&longitude=-63.39545485439191&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 13:27:29.743 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.47753281138516, lon=-63.39545485439191. Falling back to general forecast.
2025-05-26 13:27:29.743 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ais (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:30.186 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for errors (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.180 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.376 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.400 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.415 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.455 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m203.
2025-05-26 13:27:31.455 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for telemetry (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.471 UTC | INFO     | app.app                   | Inferred location for mission m203: Lat=44.661656666666666, Lon=-63.55683833333333
2025-05-26 13:27:31.471 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 13:27:31.487 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.633 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:31.694 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m203). Returning cached data from Remote: http://129.173.20.180:8086/output_past_missions/m203-SV3-1070 (C34164NS).
2025-05-26 13:27:32.204 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.661656666666666&longitude=-63.55683833333333&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 13:27:32.204 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.661656666666666, lon=-63.55683833333333. Falling back to general forecast.
2025-05-26 13:27:32.214 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 13:27:54.983 UTC | INFO     | app.app                   | Attempting local load for power (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.062 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.062 UTC | INFO     | app.app                   | Attempting local load for ctd (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.505 UTC | INFO     | app.app                   | CACHE STORE: Storing ctd for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.505 UTC | INFO     | app.app                   | Attempting local load for weather (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.586 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.586 UTC | INFO     | app.app                   | Attempting local load for waves (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.632 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.632 UTC | INFO     | app.app                   | Attempting local load for vr2c (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.667 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.667 UTC | INFO     | app.app                   | Attempting local load for solar (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.711 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.711 UTC | INFO     | app.app                   | Attempting local load for fluorometer (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.790 UTC | INFO     | app.app                   | CACHE STORE: Storing fluorometer for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.790 UTC | INFO     | app.app                   | Attempting local load for ais (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.824 UTC | INFO     | app.app                   | CACHE STORE: Storing ais for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:55.824 UTC | INFO     | app.app                   | Attempting local load for errors (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:55.844 UTC | INFO     | app.app                   | CACHE STORE: Storing errors for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:56.934 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 13:27:57.146 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 13:27:57.300 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m203.
2025-05-26 13:27:57.300 UTC | INFO     | app.app                   | Attempting local load for telemetry (mission: m203) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:27:57.360 UTC | INFO     | app.app                   | CACHE STORE: Storing telemetry for m203 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203) into cache.
2025-05-26 13:27:57.369 UTC | INFO     | app.app                   | Inferred location for mission m203: Lat=44.438475, Lon=-63.26319833
2025-05-26 13:27:57.369 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 13:27:57.384 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 13:27:57.407 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 13:27:57.440 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 13:27:57.455 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 13:27:57.476 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 13:27:58.041 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.438475&longitude=-63.26319833&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 13:27:58.041 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.438475, lon=-63.26319833. Falling back to general forecast.
2025-05-26 13:27:58.041 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 13:28:49.385 UTC | INFO     | app.app                   | Attempting local load for power (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.394 UTC | INFO     | app.app                   | CACHE STORE: Storing power for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.394 UTC | INFO     | app.app                   | Attempting local load for ctd (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.627 UTC | INFO     | app.app                   | CACHE STORE: Storing ctd for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.633 UTC | INFO     | app.app                   | Attempting local load for weather (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.659 UTC | INFO     | app.app                   | CACHE STORE: Storing weather for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.659 UTC | INFO     | app.app                   | Attempting local load for waves (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.671 UTC | INFO     | app.app                   | CACHE STORE: Storing waves for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.671 UTC | INFO     | app.app                   | Attempting local load for vr2c (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.671 UTC | INFO     | app.app                   | CACHE STORE: Storing vr2c for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.671 UTC | INFO     | app.app                   | Attempting local load for solar (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.693 UTC | INFO     | app.app                   | CACHE STORE: Storing solar for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.693 UTC | INFO     | app.app                   | Attempting local load for fluorometer (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.769 UTC | INFO     | app.app                   | CACHE STORE: Storing fluorometer for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.776 UTC | INFO     | app.app                   | Attempting local load for ais (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.788 UTC | INFO     | app.app                   | CACHE STORE: Storing ais for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:49.788 UTC | INFO     | app.app                   | Attempting local load for errors (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:49.788 UTC | INFO     | app.app                   | CACHE STORE: Storing errors for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:50.420 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m204). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:28:50.459 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m204). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:28:50.608 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m204). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:28:50.612 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m204.
2025-05-26 13:28:50.612 UTC | INFO     | app.app                   | Attempting local load for telemetry (mission: m204) from default local path: C:\Users\ty225269\Documents\Python Playground\Data
2025-05-26 13:28:50.695 UTC | INFO     | app.app                   | CACHE STORE: Storing telemetry for m204 (from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204) into cache.
2025-05-26 13:28:50.712 UTC | INFO     | app.app                   | Inferred location for mission m204: Lat=44.47753281138516, Lon=-63.39545485439191
2025-05-26 13:28:50.712 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 13:28:50.729 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m204). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:28:50.745 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m204). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:28:50.768 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m204). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:28:50.789 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m204). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m204.
2025-05-26 13:28:51.366 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.47753281138516&longitude=-63.39545485439191&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 13:28:51.366 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.47753281138516, lon=-63.39545485439191. Falling back to general forecast.
2025-05-26 13:28:51.366 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 13:53:41.923 UTC | INFO     | apscheduler.executors.default | Running job "refresh_active_mission_cache (trigger: interval[0:30:00], next run at: 2025-05-26 11:23:41 ADT)" (scheduled at 2025-05-26 10:53:41.912810-03:00)
2025-05-26 13:53:41.923 UTC | INFO     | app.app                   | BACKGROUND TASK: Starting proactive cache refresh for active real-time missions.
2025-05-26 13:53:41.923 UTC | INFO     | app.app                   | BACKGROUND TASK: Proactive cache refresh for active real-time missions completed.
2025-05-26 13:53:41.923 UTC | INFO     | apscheduler.executors.default | Job "refresh_active_mission_cache (trigger: interval[0:30:00], next run at: 2025-05-26 11:23:41 ADT)" executed successfully
2025-05-26 14:23:41.917 UTC | INFO     | apscheduler.executors.default | Running job "refresh_active_mission_cache (trigger: interval[0:30:00], next run at: 2025-05-26 11:53:41 ADT)" (scheduled at 2025-05-26 11:23:41.912810-03:00)
2025-05-26 14:23:41.917 UTC | INFO     | app.app                   | BACKGROUND TASK: Starting proactive cache refresh for active real-time missions.
2025-05-26 14:23:41.917 UTC | INFO     | app.app                   | BACKGROUND TASK: Proactive cache refresh for active real-time missions completed.
2025-05-26 14:23:41.917 UTC | INFO     | apscheduler.executors.default | Job "refresh_active_mission_cache (trigger: interval[0:30:00], next run at: 2025-05-26 11:53:41 ADT)" executed successfully
2025-05-26 14:47:57.014 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.014 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.015 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.015 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.015 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.015 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.015 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.015 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ais (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:57.015 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for errors (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:58.498 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for ctd (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:58.721 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for weather (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:58.755 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for vr2c (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:58.906 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for fluorometer (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:58.966 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for power (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:58.983 UTC | INFO     | app.app                   | Latitude/Longitude not provided for forecast. Attempting to infer from telemetry for mission m203.
2025-05-26 14:47:58.983 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for telemetry (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:58.999 UTC | INFO     | app.app                   | Inferred location for mission m203: Lat=44.438475, Lon=-63.26319833
2025-05-26 14:47:58.999 UTC | INFO     | app.core.forecast         | Attempting to fetch marine forecast from: https://api.open-meteo.com/v1/marine
2025-05-26 14:47:59.018 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for waves (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:59.039 UTC | INFO     | app.app                   | Cache hit (valid - static/local) for solar (m203). Returning cached data from Local (Default): C:\Users\ty225269\Documents\Python Playground\Data\m203.
2025-05-26 14:47:59.599 UTC | WARNING  | app.core.forecast         | HTTPStatusError when fetching https://api.open-meteo.com/v1/marine: 404 - Client error '404 Not Found' for url 'https://api.open-meteo.com/v1/marine?latitude=44.438475&longitude=-63.26319833&hourly=wave_height%2Cwave_direction%2Cwave_period%2Cwindspeed_10m%2Cwinddirection_10m%2Cocean_current_velocity%2Cocean_current_direction%2Cweathercode%2Ctemperature_2m%2Cprecipitation&timezone=GMT'
For more information check: https://httpstatuses.com/404
2025-05-26 14:47:59.599 UTC | WARNING  | app.core.forecast         | Marine forecast API returned 404 for lat=44.438475, lon=-63.26319833. Falling back to general forecast.
2025-05-26 14:47:59.599 UTC | INFO     | app.core.forecast         | Attempting to fetch general forecast from: https://api.open-meteo.com/v1/forecast
2025-05-26 14:53:41.919 UTC | INFO     | apscheduler.executors.default | Running job "refresh_active_mission_cache (trigger: interval[0:30:00], next run at: 2025-05-26 12:23:41 ADT)" (scheduled at 2025-05-26 11:53:41.912810-03:00)
2025-05-26 14:53:41.920 UTC | INFO     | app.app                   | BACKGROUND TASK: Starting proactive cache refresh for active real-time missions.
2025-05-26 14:53:41.920 UTC | INFO     | app.app                   | BACKGROUND TASK: Proactive cache refresh for active real-time missions completed.
2025-05-26 14:53:41.920 UTC | INFO     | apscheduler.executors.default | Job "refresh_active_mission_cache (trigger: interval[0:30:00], next run at: 2025-05-26 12:23:41 ADT)" executed successfully
2025-05-26 15:01:47.166 UTC | INFO     | app.app                   | APScheduler shut down.
2025-05-26 15:01:47.166 UTC | INFO     | apscheduler.scheduler     | Scheduler has been shut down
